pom.xml

```diff
@@ -10,7 +10,7 @@
     <dependency>
       <groupId>org.clojure</groupId>
       <artifactId>clojure</artifactId>
-      <version>1.10.1</version>
+      <version>1.10.0</version>
```

Comment:
Hi,

1.10.0 has a serious regression that causes massive performance problems during startup because of changes to Java itself. This was fixed in 1.10.1. More details here https://clojure.org/news/2019/06/06/clojure1-10-1.